graphdatascience/model/model.py

```diff
@@ -55,3 +56,24 @@ def drop(self) -> None:
         params = {"model_name": self._name}
 
         self._query_runner.run_query(query, params)
+
+    @abstractmethod
+    def _query_prefix(self) -> str:
+        pass
+
+    def metrics(self) -> Dict[str, Any]:
+        return self._list_info()["modelInfo"]["metrics"]  # type: ignore
+
+    def predict_stream(self, G: Graph, **config: Any) -> QueryResult:
+        query = f"{self._query_prefix()}stream($graph_name, $config)"
+        config["modelName"] = self.name()
+        params = {"graph_name": G.name(), "config": config}
```

Comment:
it's a bit unfortunate to have to mix snake case and camel case conventions. is the user required to do that also, or are we making it possible for them to stay "pythonic" all the way?